pos_order_return/models/pos_order.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2018 Sylvain LE GAL (https://twitter.com/legalsylvain)
+# Copyright 2018 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import _, api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.exceptions import ValidationError
+
+
+class PosOrder(models.Model):
+    _inherit = 'pos.order'
+
+    returned_order_id = fields.Many2one(
+        comodel_name='pos.order',
+        string='Returned Order',
+        readonly=True,
+    )
+    refund_order_ids = fields.One2many(
+        comodel_name='pos.order',
+        inverse_name='returned_order_id',
+        string='Refund Orders',
+        readonly=True,
+    )
+    refund_order_qty = fields.Integer(
+        compute='_compute_refund_order_qty',
+        string='Refund Orders Quantity',
+        digits=dp.get_precision('Product Unit of Measure'),
+    )
+
+    def _compute_refund_order_qty(self):
+        for order in self:
+            order.refund_order_qty = len(order.refund_order_ids)
```

Comment:
This can be best performance implemented with a `read_group` and using count returned value inside the tuple.

Reply:
Do you have an example of it?